parameter/ParameterMgr.cpp

```diff
@@ -941,7 +940,7 @@ CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::statusCommandProces
 
     /// Criteria states
     appendTitle(strResult, "Selection Criteria:");
-    list<string> lstrSelectionCriteria;
+    core::Results lstrSelectionCriteria;
```

Comment:
in most places, you've removed hungarian notation when using "Results". you should do it here too.